### ISSUE TITLE
Remaster Lighting on Bronco Stadium Day

### ISSUE THREAD

**Issue by timblegoorn** [cs481-ekh/f22-kitty-cat-studios#335]
After working on the night lighting for Bronco Stadium Night, I will work on remastering the day lighting to bring it up to a similar quality and style as the night map.

**Comment by timblegoorn** [comment]
This is how the daytime lighting looked before any changes I made:

![image](https://user-images.githubusercontent.com/43420737/200149696-d2cf124e-ca3d-49e7-b5e5-e8e63dee28ff.png)

**Comment by timblegoorn** [comment]
The biggest problem is the stadium stands used emissive materials for the orange and blue, which caused the harsh lighting. In addition, the lighting for the stands just flat out doesn't work as static object. Thus, they were changed to stationary objects, which is a lower performance but shouldn't matter too much in this case. (I had to do the same for night map) I also changed the fog settings a bit and added some clouds and adjusted the sky and overall temperature of lighting.

Here is the result after my changes:
![HighresScreenshot00000](https://user-images.githubusercontent.com/43420737/200149701-73c43a7d-ed8c-49ec-aef5-0587aea4ccce.png)
